src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -15,18 +16,24 @@ public static long countWords() throws IOException {
                 .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
-        long count = 0;
-        for (String w : words) {
-            if (w.length() > 12) count++;
-        }
-        return count;
+        return words.stream()
+                    .filter(w -> w.length() > 12)
+                    .count();
     }
 
     public static void printLongestWordTop100() throws IOException {
         String contents = new String(Files.readAllBytes(Paths
                 .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
+        words.stream()
+                .filter(word -> word.length() > 12)
+                .sorted(Comparator.comparingInt(String::length).reversed())
+                .distinct()
```

Comment:
```suggestion
                .distinct()
                .sorted(Comparator.comparingInt(String::length).reversed())
```

지금은 별 차이 없지만, 데이터가 많을 경우 distinct() 후에 sorted()하는 것과 성능 차이가 있을 수 있어요.